examples/simple/package.json

```diff
@@ -31,8 +29,10 @@
         "react-router-dom": "^6.22.0"
     },
     "devDependencies": {
-        "@hookform/devtools": "^4.0.2",
+        "@hookform/devtools": "^4.3.3",
         "@vitejs/plugin-react": "^4.2.1",
+        "little-state-machine": "^4.8.1",
+        "react-simple-animate": "^3.5.3",
```

Comment:
Had to add those manually (hookform devtools depends on them) to have react-19 support